distribute/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# makepkg is available in bullseye's backports (and newer distros) only
+# checkinstall is available in bullseye only, too
+FROM debian:bullseye-backports
+
+SHELL ["bash", "-x", "-c"]
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# note: don't build with OpenJDK > 16, as this is the runtime we ship
```

Comment:
Our DEB package indicates that it is also compatible with Java 11:
`--requires 'bash,openjdk-11-jre\|openjdk-17-jre,potrace' \`

Java supports backwards-compatible compilation, depending on some setting in the maven file. https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html

Reply:
Please feel free to change this. I don't see a reason not to use OpenJDK 11 at this point, as it is still a supported release as far as I can see.